packages/nextlove/src/types/index.ts

```diff
@@ -35,6 +35,7 @@ export interface RouteSpec<
   middlewares?: Middlewares
   jsonResponse?: JsonResponse
   formData?: FormData
+  sdkReturnValue?: string
```

Comment:
what would be the expected values for this?

jsdoc comment with usage would be helpful

Reply:
Some API endpoints may return an object with multiple keys. The code generator can select a key from the response using the sdkReturnValue. 

For example, 
/locks/get returns lock and device. If the user adds the sdkReturnValue with the value device, the SDK will automatically understand and return the device directly.

Reply:
@codetheweb just added the comments ;)